api/nakadi-event-bus-api.yaml

```diff
@@ -613,6 +632,11 @@ paths:
             The flow id of the request, which is written into the logs and passed to called services. Helpful
             for operational troubleshooting and log analysis.
           type: string
+        - name: offset
+          in: parameter
+          description: |
+            Offset to query for unconsumed events. Depends on `partition` parameter. When present adds the property
+            `unconsumed_events` to the response partition object.
```

Comment:
Maybe it is better to change name to `consumed_offset`, so it will hold some information about which offset to use.